dependencies.yaml

```
@@ -111,6 +119,14 @@ files:
       - depends_on_libcuvs
       - depends_on_libraft
       - depends_on_nccl
+  java:
+    output: none
+    includes:
+      - cuda
+      - cuda_version
+      - rapids_build
+      - depends_on_libcuvs
+      - java
```

```suggestion
      - cuda
      - cuda_version
      - depends_on_libcuvs
      - depends_on_libraft
      - depends_on_nccl
      - java
      - rapids_build
```

For my own understanding... why is it that the Go and Rust bindings need `libraft` and NCCL but the Java bindings do not? If the answer is "the Java bindings do need `libraft` and NCCL and must just be getting those transitively via `libcuvs`", then please add explicit dependencies here. That'll ensure we get the correct versions of those things.

Also, please keep these lists alphabetized. That makes them a bit easier for people to read, reducing the risk of mistakes.

---

Java bindings do need libraft (and NCCL, becaue cuVS requires NCCL).

---

Great, thank you. I'm glad NCCL and libraft were added here in https://github.com/rapidsai/cuvs/pull/805/commits/88165ad82833f3dabdfb90c0d8ca59196d2f4c7a